components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -75,6 +75,12 @@ bool GetUseStagingInfuraEndpoint() {
   return env->HasVar("BRAVE_INFURA_STAGING");
 }
 
+#if defined(OS_WIN) || defined(OS_MAC)
+const char kGanacheLocalhostURL[] = "http://localhost:7545/";
+#else
+const char kGanacheLocalhostURL[] = "http://localhost:8545/";
+#endif
```

Comment:
@AlexeyBarabash double checked on my side Ganache v2.5.4 has 7545 port for linux too, can you take latest version and recheck it for you?

Reply:
I leave it as constant to be easier to change it everywhere if you have specific config

Reply:
@spylogsster 
You are right, port 8545 is for an obsolete version of Ganace (ganache-core: 2.13.2), which was installed by `npm install -g ganache-cli`.

The actual version downloaded from https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/ganache-2.5.4-linux-x86_64.AppImage, `v2.5.4` uses port `7545` by default.

Sorry for troubles.